### PR TITLE
EDUCATOR-3944 Add is_active field to curriculum membership models

### PR DESCRIPTION
**Pull request by kdmccormick**
**Ticket:** https://openedx.atlassian.net/browse/EDUCATOR-3944
**Team:** @edx/educator-neem

### CODE REVIEW

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -1700,6 +1701,7 @@ class CurriculumCourseMembership(TimeStampedModel):
     """
     curriculum = models.ForeignKey(Curriculum, on_delete=models.CASCADE)
     course = models.ForeignKey(Course, on_delete=models.CASCADE)
+    is_active = models.BooleanField(default=False)
```
Comment (schenedx) [inline course_discovery/apps/course_metadata/models.py]: I think we should make `default=True` for both models. The rational is, during the first setup, the intention is definitely to make the membership count. But then we should leave the option for anyway to toggle it off later.

**Review by schenedx — CHANGES_REQUESTED** [review]
Please have the default to be True

File: course_discovery/apps/course_metadata/models.py (hunk 1 above)
Comment (kdmccormick) [reply course_discovery/apps/course_metadata/models.py]: Sounds good :+1:

**Review by schenedx — APPROVED** [review]
👍

**Pull request merged**